sht31-controller/Support.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE NamedFieldPuns #-}
+module Support (subscribe, persist) where
+
+import Control.Monad (void)
+import Network.AMQP as AMQP
+import System.Environment (getEnv)
+import qualified Data.ByteString.Lazy as BL
+import qualified Data.Text as T
+
+e key = T.pack <$> getEnv key
+
+subscribe respondFunc = do
+    h <- getEnv "HOST"
+    u <- e "USER"
+    p <- e "PASS"
+    exchangeName <- e "EXCHANGE"
+    conn <- AMQP.openConnection h "/" u p
+    chan <- AMQP.openChannel conn
+
+    (queueName, _, _) <- AMQP.declareQueue chan AMQP.newQueue {queueAutoDelete = True}
+    AMQP.declareExchange chan AMQP.newExchange {exchangeName, exchangeType = "topic"}
+    AMQP.bindQueue chan queueName exchangeName "controller.44"
```

Comment:
The value after `controller.` must be a parameter. This parameter should be a list so if multiple addresses are provided (e.g. `0x44`, `0x45`), both addresses are mapped to a routing key, binding to the queue.